doc/en/user/source/community/gwc-sqlite/index.rst

```diff
@@ -0,0 +1,158 @@
+.. _community_gwc_sqlite:
+
+GWC SQLite Plugin
+=================
+
+This plugin provides integration with GWC SQLite based blob stores. At the moment only one blob store of this type is available, the MBTiles blob store.
+
+MBTiles BlobStore
+-----------------
+
+This blob store allow us to store tiles using the `MBTiles <https://github.com/mapbox/mbtiles-spec/blob/master/1.1/spec.md>`_ specification (version 1.1) which defines a schema for storing tiles in an `SQLite <https://www.sqlite.org/>`_ database with some restrictions regarding tiles formats and projections.
+
+MBTiles specification only supports JPEG and PNG formats and projection EPSG:3857 is assumed. The implemented blob store will read and write MBTiles files compliant with the specification but will also be able to write and read MBTiles files that use others formats and projections.
+
+Using the MBTiles blob store will bring several benefits at the cost of some performance loss. We will be able to store our data with a significantly smaller number of files, which will help us avoid file system headaches and help us managing our data. In some cases the stored data will be more compacted reducing the size of the data on disk.
+
+When compared to the file blob store this store as two limitations:
+
+* This store will not integrate with disk quota, this is a consequence of using database files.
```

Comment:
Does not integrate... how? Like, is disk quota disabling itself, ignoring the storage? Or it's just that if enabled the stats kept in disk quota do not make much sense and do not reflect the actual disk usage?
If it's the latter, I'd clarify and state that it's best not to enable disk quota because the size of the dbs cannot be really controlled.